SF iOS/SF iOS/Feed/SwiftUI/FeedView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  FeedView.swift
+//  Coffup
+//
+//  Created by Roderic Campbell on 6/8/19.
+//
+
+import SwiftUI
+
+struct FeedView : View {
+    let myEvents: [WrappedEvent]
+    var body: some View {
+        NavigationView {
+            List(eventData.identified(by: \.eventID)) { someEvent in
+                NavigationButton(destination: EventDetailView(event: someEvent)) {
+                    FeedRow(event: someEvent)
+                }
+            }
+        }
+            .navigationBarTitle(Text("Events"))
+            .navigationBarHidden(false)
+    }
+}
```

Comment:
`FeedView` is where it all starts

Reply:
Can't wait for a SwiftUI style guide on how to order your views in a project.